File: consultoria/src/main/java/br/com/sigo/consultoria/security/dto/TokenDto.java
package br.com.sigo.consultoria.security.dto;

import br.com.sigo.consultoria.enums.PerfilEnum;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class TokenDto {

  private String token;

  private String username;

  private String displayName;

  private List<PerfilEnum> perfis;
}

File: consultoria-app/app/src/main/java/br/com/sigo/consultoria/dtos/TokenDto.kt
package br.com.sigo.consultoria.dtos

import br.com.sigo.consultoria.enums.PerfilEnum
import java.io.Serializable

data class TokenDto(
    var token: String,
    var username: String,
    var displayName: String,
    var perfis: List<PerfilEnum>
) : Serializable
